clio-server/src/main/scala/org/broadinstitute/clio/server/dataaccess/HttpElasticsearchDAO.scala

```diff
@@ -47,23 +47,41 @@ class HttpElasticsearchDAO private[dataaccess] (
     HttpClient.fromRestClient(restClient)
   }
 
-  override def updateMetadata(documents: Json*)(
+  override def updateMetadata(documents: Seq[Json])(
     implicit index: ElasticsearchIndex[_]
   ): Future[Unit] = bulkUpdate(documents.map(updatePartialDocument(index, _)))
 
-  override def queryMetadata(queryDefinition: QueryDefinition)(
+  override def updateMetadata(document: Json)(
+    implicit index: ElasticsearchIndex[_]
+  ): Future[Unit] = updateMetadata(Seq(document))
+
+  override def rawQuery(json: JsonObject)(
     implicit index: ElasticsearchIndex[_]
   ): Source[Json, NotUsed] = {
+    val requestBody =
+      json
+        .add(
+          HttpElasticsearchDAO.DocumentScrollSizeName,
+          HttpElasticsearchDAO.DocumentScrollSize.asJson
+        )
+        .add(
+          HttpElasticsearchDAO.DocumentScrollSortName,
+          HttpElasticsearchDAO.DocumentScrollSort.asJson
+        )
+        .asJson
+        .pretty(ModelAutoDerivation.defaultPrinter)
     val searchDefinition = searchWithType(index.indexName / index.indexType)
       .scroll(HttpElasticsearchDAO.DocumentScrollKeepAlive)
-      .size(HttpElasticsearchDAO.DocumentScrollSize)
-      .sortByFieldAsc(HttpElasticsearchDAO.DocumentScrollSort)
-      .query(queryDefinition)
+      .source(requestBody)
 
     val responsePublisher = httpClient.publisher(searchDefinition)
     Source
       .fromPublisher(responsePublisher)
       .map(_.to[Json])
+      .alsoTo(
+        Sink
+          .foreach(json => logger.debug(json.pretty(ModelAutoDerivation.defaultPrinter)))
```

Comment:
Does it work as `foreach(logger.debug(_.pretty(...)))`?

Reply:
doesn't seem to